git-scripts/git-vgrep

```diff
@@ -3,8 +3,22 @@
 set -e
 out=$(mktemp -t git-vgrep-XXXXXX)
 trap "rm -f $out" EXIT
-git grep -n "$@" | tee "$out"
-if [ "$(cat "$out" | wc -l)" -eq 1 ] ; then
+git grep -n "$@" | tee "$out" 1> /dev/null
+
+lines=$(cat -b "$out" | wc -l)
+
+if [ $lines -eq 0 ]; then
+	exit 0
+fi
+
+if [ $lines -eq 1 ] ; then
```

Comment:
This logic could be combined. I'd do:

```
if [ $line -ne 1 ]; then
    LINE=$(awk "NR==$line" "$out")
    echo "$LINE" > "$out"
fi
```

and then the rest Just Works.

Reply:
Much better! Done.